scripts/istat/geos/import_config.txt

```diff
@@ -0,0 +1,41 @@
+import: {
+  import_name: "istat-region"
+  provenance_url: "https://www.istat.it/it/archivio/6789"
+  provenance_description: "ISTAT code for territorial units in Italy"
+  mcf_url: "https://storage.cloud.google.com/resolved_mcf/external_tables/istat-region/nodes.mcf-00000-of-00001"
+  # First date of imported data.
+  start_date: "2020-06-23"
+  # Last date of imported data.
+  end_date: "2020-06-23"
+  # Source data release schedule.
+  update_frequency: "P1Y"
```

Comment:
Wow, do they really release new codes yearly?

Reply:
According to their previous archives, they update it once/twice a year. Looks like it's mostly small changes in municipal names. Regarding the code, it's updated less frequently. The current version is adopted since 2017. The previous versions are (2010-2016), (2006-2009), (1995-2005).